Gruntfile.js

```diff
@@ -14,7 +15,7 @@ module.exports = function (grunt) {
         // Metadata.
         meta: {
             version: {
-                eva: '3.1.0'
+                eva: '4.0.0'
```

Comment:
This applies to the migration to 4.0 but doesn't mean that every code in develop will become 4.0 right now.